cdap-docs/_common/_themes/cdap/static/cdap.css_t

```diff
@@ -214,17 +200,34 @@ div.body h4 { background-color: #EDEDED; margin: 30px 0px 10px 0px; }
 div.body h5 { background-color: #F2F2F2; margin: 30px 0px 10px 0px; }
 div.body h6 { background-color: #F2F2F2; }
 
+a {
+    color: #ff6600;
+    text-decoration: none;
+}
+
+a.current {
+    font-weight: bold;
+}
+
+a:hover {
+    color: #005B81;
+    text-decoration: underline;
```

Comment:
The default behavior is that links are underlined, both active and on hover. We have been using a style of having active links be *not* underlined, but instead be in bold and in the corporate color (originally blue, now orange). The reason for not underlining was that on a page of text with many links inline, it was far too distracting and busy to read. 

Instead, by eliminating the underlines except on hovering, it was cleaner look for text runs.